packages/commonwealth/server/controllers/server_communities_methods/get_communities.ts

```diff
@@ -18,6 +18,14 @@ export async function __getCommunities(
     {
       model: this.models.CommunityStake,
     },
+    {
+      model: this.models.CommunityTags,
```

Comment:
Also we should have an index on CommunityTags.community_id. Otherwise this will be a sequential scan instead of a index scan (O(n) vs O(logn)).